client/src/shared-modules/Footer.js

```diff
@@ -1,11 +1,76 @@
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable linebreak-style */
+
 import React from "react";
+import { useState, useEffect } from "react";
+
+import "../pages/Footer.css";
 
 function Footer() {
+
+  const [pages, setPages] = useState([]);
+
+	async function fectPages() {
+		try {
+			const response = await fetch(
+				"https://starter-kit-slji.onrender.com/api/pages"
+			);
+			const pagesFromDatabase = await response.json();
+			setPages(pagesFromDatabase);
+		} catch (err) {
+			console.error(err);
+		}
+	}
+
+	useEffect(() => {
+		fectPages();
+	}, []);
+
+
+
   return (
-    <div>
-      <h1>Footer place holder</h1>
-    </div>
-  );
+		<div className="card-footer">
+			<section className="footer-address-section">
+				<img src="#" alt="The Collective Foundation logo" />
+				<h3>The Collective Foundation</h3>
+				<ul>
+					<li>Address</li>
+					<li>City & Postcod</li>
+					<li>Contact number</li>
+				</ul>
+			</section>
+
+			<section className="footer-pageAndSocialmedia-section">
+				<div className="footer-pages">
+					<h2>Our Work</h2>
+          <ul>
+            {/* this is looping throught the pages and creating each back a botton in the footer */}
+						{pages.map((page) => (
```

Comment:
I cannot see it's working, please see the attached, please make sure you have connected to the DB properly.
![Screenshot from 2023-03-13 20-50-24](https://user-images.githubusercontent.com/107772913/224829710-3de03501-86b7-4d5d-a16a-c254087b6533.png)